src/obstacle/MeshObstacle.cxx

```diff
@@ -484,7 +485,8 @@ bool MeshObstacle::containsPointNoOctree(const float point[3]) const
     {
         if (checkTypes[c] == CheckInside)
         {
-            vec3sub (dir, checkPoints[c], point);
+            for (int i = 0; i < 3; i++)
+                dir[i] = checkPoints[c][i] - point[i];
```

Comment:
this seems unfortunate. Perhaps you got tired of permuting the interfaces? Would this have been easier if `point` were changed to `glm::vec3`?